src/jobs/windows.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Windows Engineer
+toc: false
+---
+
+## About the team
+
+Flutter is the most popular multi-platform UI toolkit, with shipped support for iOS, Android, Web, Windows, macOS, and Linux. On Windows, we're allowing Flutter developers to target an additional 1B users, and we're offering a better alternative to desktop developers who use Electron, Windows Forms, WPF, or (shudder) UWP!
+
+The Flutter Desktop Team is responsible for the desktop-specific (Windows, macOS, Linux) embeddings of the Flutter engine, which includes enabling platform plugins, nesting platform views inside Flutter apps, and wiring-up accessibility hooks on those platforms – and anything else that helps Flutter feel “native” when run in those environments.
+
+## About the position
+
+In the "Windows Engineer" role you'll be part of the team that makes Flutter apps the absolute best they can be when running on Windows, including:
+
+*   Designing and building multi-window support.
+*   Building support for Windows system menu integration.
+*   Adding support for shell / window drag ‘n drop.
+*   Adding support for nesting Windows UI views inside Flutter apps.
+*   Delivering outstanding performance on Windows.
+*   Nurturing a thriving community of contributions from open-source developers by reviewing GitHub issues and pull requests (PRs).
+
+## Our values
+
+### Mentorship
+
+Upon joining Google, you will be paired with a formal mentor, who will help guide you in the process of ramping up, forging relationships, and learning the systems you’ll need to do your job.  Your manager can also help you find mentors who can coach you as you navigate your career at Google. In addition to formal mentors, we work and train together so that we are always learning from one another, and we celebrate and support the career progression of our team members.
```

Comment:
```suggestion
Upon joining Google, you will be paired with a formal mentor,
who will help guide you in the process of ramping up, forging relationships,
and learning the systems you’ll need to do your job. Your manager can also
help you find mentors who can coach you as you navigate your career at Google.
In addition to formal mentors, we work and train together so that we are
always learning from one another, and we celebrate and support the career progression
of our team members.
```